src/main/java/org/killbill/billing/plugin/analytics/AnalyticsListener.java

```diff
@@ -304,6 +310,20 @@ private boolean jobsOverlap(final AnalyticsJob job, final NotificationEventWithM
     }
 
     private void handleAnalyticsJob(final AnalyticsJob job) throws AnalyticsRefreshException {
+        GlobalLock lock = null;
+        try {
+            lock = locker.lockWithNumberOfTries("ANALYTICS_REFRESH", job.getAccountId().toString(), 100);
```

Comment:
Nit: `ANALYTICS_REFRESH` -> `ANALYTICS_REFRESH-`

Reply:
This is already done behind the scenes: https://github.com/killbill/killbill-commons/blob/5d736e923eb1b3e951def9b9f8995d4936e7e562/locker/src/main/java/org/killbill/commons/locker/mysql/MySqlGlobalLocker.java#L47

Reply:
👍